Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -55,9 +56,28 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
       .sorted()
       .map { (project, Either<Reward, Backing>.left($0)) }
 
+    let availableRewards = project.rewards
+      .filter { isMainReward(reward: $0, project: project) }
+      .filter { $0.remaining != 0 }
```

Comment:
i think we should also add a check for if remaining is `nil` here, since it's an optional value and if it is nil the rewards are unlimited. this filter can then be more like:

```swift
.filter { $0.remaining == nil || $0.remaining != 0 }
```